doc/log.rst

```diff
@@ -20,28 +20,32 @@ given as "watches" are also periodically logged to the console.
 To analyze this data, we need some tools that are distributed as part of this
 package.
 
-To prepare this data set for analysis, run ``runalyzer-gather`` as a script::
+You may use ``runalyzer`` to analyze the log file::
+
```

Comment:
A message should be sent notifying the user that an in-memory db has been created